pom.xml

```diff
@@ -167,8 +157,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>8</source>
+                    <target>8</target>
                     <encoding>UTF-8</encoding>
```

Comment:
Please revert the Java version and modify the lambda function

Reply:
resolved